### PR TITLE
Remove py26reqs.txt

### PR DESCRIPTION
**Pull request by mgedmin**
ConfigArgParse 0.10 from PyPI supports Python 2.6, so there's no more
need to install a fixed version directly from a git branch.

### CODE REVIEW

**Comment by bmw** [comment]
LGTM. Thanks for noticing this @mgedmin. I didn't realize that @kuba's changes had landed in the official tree.

I'll leave this up to @pde on whether we want to make this change before public beta.

**Comment by kuba** [comment]
0.10 was released without fixing https://github.com/bw2/ConfigArgParse/issues/26? :( (Un)luckily, we stopped using subparsers, so we should not be affected.

`setup.py` should be updated to `argaprse>0.10` in case of py26

File: bootstrap/README [hunk 1]
```diff
@@ -2,6 +2,6 @@ This directory contains scripts that install necessary OS-specific
 prerequisite dependencies (see docs/using.rst).
 
 General dependencies:
-- git-core: py26reqs.txt git+https://*
```
Comment (kuba) [inline bootstrap/README]: if this is removed, then `git-core` should be removed as well, or the comment should be updated if, some other projects are downloaded using `pip install git+http...`

**Comment by mgedmin** [comment]
> setup.py should be updated to argaprse>0.10 in case of py26

I assume you meant ConfigArgParse>=0.10.0?

File: bootstrap/README (hunk 1 above)
Comment (kuba) [inline bootstrap/README]: FYI, I partially meant removal from deps in bootstrap scripts

File: setup.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def read_file(filename, encoding='utf8'):
 
 install_requires = [
     'acme=={0}'.format(version),
-    'ConfigArgParse',
+    'ConfigArgParse>=0.10.0',  # python2.6 support, upstream #17
```
Comment (kuba) [inline setup.py]: Version pin should be moved to else branch of sys.version_info < (2, 7) condition checked below.

Comment (mgedmin) [inline setup.py]: Hmm.  Dynamically-computed `install_requires` causes problems for users of pip >= 7.0, which has wheel caching enabled by default: https://github.com/pypa/pip/issues/3025.  Short version:
- I pip install letsencrypt on Python 2.7, pip runs setup.py and builds a wheel (marked -py2-none-any.whl) and puts it in ~/.cache/pip/
- I pip install letsencrypt on Python 2.6, pip discovers a valid wheel and installs it, using `install_requires` computed earlier for a different Python version.

What's the harm of having everyone use a recent version of ConfigArgParse?

**Comment by mgedmin** [comment]
Rebased to resolve merge conflict in Dockerfile-dev.

File: setup.py (hunk 2 above)
Comment (kuba) [inline setup.py]: Well, if that causes the problem, then please open a separate bug to discuss changing our strategy. But this PR is not the best place to discuss is, or deviate from already established standards.

One potential harm that instantly comes to my mind is that it makes OS packaging a bit harder (maintainers, such as @hlieberman, will have to patch `setup.py` unnecessarily).

**Comment by mgedmin** [comment]
#1908 supersedes this.

**Pull request closed without merging**